src/helpers/cookie.helper.js

```diff
@@ -6,28 +6,17 @@ const { COOKIES } = require('app.constants');
 exports.setTokenCookies = ({
   ctx,
   accessToken,
-  accessTokenExpireAt,
-  refreshToken,
-  refreshTokenExpireAt,
 }) => {
   const parsedUrl = url.parse(config.webUrl);
   const parsed = psl.parse(parsedUrl.hostname);
   const cookiesDomain = parsed.domain;
 
   ctx.cookies.set(COOKIES.ACCESS_TOKEN, accessToken, {
-    httpOnly: false,
-    expires: accessTokenExpireAt,
```

Comment:
Do not cookies without an expiration time behave like session time cookies?

Reply:
fixed